### DISCUSSION TITLE
Currently unable to use vernier in a serial code

### DISCUSSION THREAD

**Discussion by mo-andymalcolm** [MetOffice/Vernier#164] [Ideas]
Createbc is a serial only code which is currently profiled using another tool.

We wish to use vernier to profile it going forward.

The problem we have is that vernier_init requires mpi_comm_world_v and also that vernier requires mpi_init to have been run to work at all. 

Possible solution is to try and introduce a serial version of vernier. Other suggestions welcomed.

**Comment by andrewcoughtrie** [comment]
It has been agreed that the short term solution to this is to have a version of Vernier installed that doesn't link to the MPI library, this is the current mechanism built in to allow profiling of code that doesn't initialise MPI.
